src/core/radio/max_power_table.hpp

```diff
@@ -26,53 +26,51 @@
  *  POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef OT_POSIX_PLATFORM_MAX_POWER_TABLE_HPP_
-#define OT_POSIX_PLATFORM_MAX_POWER_TABLE_HPP_
+#ifndef OT_CORE_RADIO_MAX_POWER_TABLE_HPP_
+#define OT_CORE_RADIO_MAX_POWER_TABLE_HPP_
 
 #include "core/radio/radio.hpp"
+#include "openthread/platform/radio.h"
 
 namespace ot {
-namespace Posix {
 
 class MaxPowerTable
 {
 public:
     static const int8_t kPowerDefault = 30; ///< Default power 1 watt (30 dBm).
 
-    MaxPowerTable(void) { memset(mPowerTable, kPowerForbidden, sizeof(mPowerTable)); }
+    MaxPowerTable(void) { memset(mPowerTable, kPowerDefault, sizeof(mPowerTable)); }
 
     /**
-     * This method gets the max allowed transmit power of channel @p aChannel.
+     * This method gets the max supported transmit power of channel @p aChannel.
      *
      * @params[in]  aChannel    The radio channel number.
      *
-     * @returns The max allowed transmit power in dBm.
+     * @returns The max supported transmit power in dBm.
      *
      */
     int8_t GetTransmitPower(uint8_t aChannel) const { return mPowerTable[aChannel - Radio::kChannelMin]; }
 
     /**
-     * This method sets the max allowed transmit power of channel @p aChannel.
+     * This method sets the max supported transmit power of channel @p aChannel.
```

Comment:
Is `allowed` more descriptive?
```suggestion
     * This method sets the max allowed transmit power of channel @p aChannel.
```